CASSANDRA-18606 Fix leftnav cross-references

polandll opened the pull request:
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

jacek-lewandowski committed:
Upgraded to Netty 4.1.96

- Add Bouncycastle dependency
- Upgrade tcnative boringssl
- Add TLSv1.3 to encryption options tests
- Revert defaults after changes in Netty 4.1.75
- Remove Guava 18 from deps - we accidentally ended with Guava 30+ and 18 on the classpath because JimFS includes it as a transient dependency.

Patch by Jacek Lewandowski and Brandon Williams; reviewed by Ekaterina Dimitrova and Berenguer Blasi for CASSANDRA-17992

Co-authored-by: Jacek Lewandowski <lewandowski.jacek@gmail.com>
Co-authored-by: Brandon Williams <driftx@gmail.com>

polandll committed:
fix nav xrefs

polandll committed:
fix nav xrefs

polandll committed:
Fix lefnav cross-references in the docs

patch by Lorina Poland; reviewed by Mick Semb Wever for CASSANDRA-18606

polandll committed:
Fix lefnav cross-references in the docs

patch by Lorina Poland; reviewed by Mick Semb Wever for CASSANDRA-18606

polandll committed:
fix leftnav xrefs

polandll committed:
Merge branch 'CASS-18606' of github.com:polandll/cassandra into CASS-18606

michaelsembwever commented:
to test i run
```
./run.sh website container -a BUILD_USER_ARG:$(whoami) -a UID_ARG:$(id -u) -a GID_ARG:$(id -g)
./run.sh website build -g -b cassandra:CASS-18606 -u cassandra:https://github.com/polandll/cassandra
```
(on macos i needed some small fixes, pushed [here](https://github.com/apache/cassandra-website/commit/466d6ffe4624b4a4a09b9946472045a7d48097ea))

Via the navigation i can now reach the new pages.

An example of a new page: `cassandra-website/content/doc/trunk/cassandra/managing/operating/virtualtables.html` 
and an example of the old page: `cassandra-website/content/doc/trunk/cassandra/managing/virtualtables.html`

Side question, for this page why do both still exist ? 

Another side question, the navigation does not visually structure as the IA (and underlying files) do.  Is that intentional?
Screenshot of the nav bar 
<img width="388" alt="Screenshot 2023-06-29 at 11 47 35" src="https://github.com/apache/cassandra/assets/559444/423a857a-b752-4a93-8ff6-41292fb07066">

polandll committed:
fix duplicate of virtualtables file

polandll committed:
reorganize left nav correctly

polandll committed:
reorganize left nav correctly

polandll committed:
still working lefthand nav

polandll committed:
still working lefthand nav

polandll committed:
fix path error

polandll committed:
fix path error

polandll committed:
move tools

Mmuzaf committed:
Fix HandshakeTest initiated twice for the outbound connection

patch by Maxim Muzafarov; reviewed by Jon Meredith and Mchael Semb Wever for CASSANDRA-18704

polandll committed:
cass-stress change

polandll committed:
cass-stress change

polandll committed:
Merge branch 'trunk' into CASS-18606

polandll commented:
@michaelsembwever Unbeknownst to me, someone else wrote `cassandra-website/content/doc/trunk/cassandra/managing/virtualtables.adoc` back into the repo. I'm not quite sure how to get rid of it, because, of course, it doesn't exist in my branch, so I can't "delete" it.

polandll committed:
add changes to virtualtables.adoc

polandll committed:
add changes to virtualtables.adoc

polandll committed:
deal with virtualtables.adoc moved file

polandll committed:
fix conflicts

polandll committed:
fix conflicts

polandll committed:
fix conflict

polandll commented:
Closing and reopening due to merge conflicts.
